sources/core/objects/Human.cpp

```diff
@@ -509,7 +529,7 @@ void Human::Bump(IdPtr<IMovable> item)
 
         unsigned int blood_value = (GetRand() % 7) + 1;
 
-        if (IdPtr<Floor> floor = GetTurf())
+        if (IdPtr<Floor> floor = GetTurf()) // TODO Bleeding only with ballistic bullets
```

Comment:
There should be functions `AddBruteDamage`, `AddBurnDamage`, `AddSuffocationDamage`, so `brute_damage_` and etc will not be used directly.
It will allow to implement bleeding on any brute damage (just code will be put in the `AddBruteDamage` function).